### DISCUSSION TITLE
When will tiktoken 0.7.0 dependency upgrade be released?

### DISCUSSION THREAD

**Discussion by skoryky** [microsoft/teams-sdk#1788] [Q&A] [answered]
When will the change to upgrade the tiktoken dependency to version 0.7.0 be released? It was committed about a week ago by @aacebo in 
https://github.com/microsoft/teams-ai/commit/9ab23f5a249b62ca3697a33b396151f1cc013476

My project requires tiktoken 0.7.0, so I can't easily use teams-ai 1.2.1.

**Comment by corinagum** [comment, marked as answer]
Python patch was just released!
